config/routes.rb

```diff
@@ -17,24 +17,24 @@
   get '/people/members', to: 'people#members'
   get '/people/members/current', to: 'people#current_members'
   get '/people/a-z', to: 'application#a_to_z', as: 'people_a_z'
-  match '/people/a-z/:letter', to: 'people#letters', letter: /[a-z]/, via: [:get], as: 'people_a_z_letter'
+  match '/people/a-z/:letter', to: 'people#letters', letter: /[A-Za-z]/, via: [:get], as: 'people_a_z_letter'
```

Comment:
I added the uppercase after talking with @aebirim as she felt it was confusing.  However this is now redundant as these a-z pages will no longer be restricted to a single letter anyway.